packages/portalnetwork/src/wire/utp/Utils/math.ts

```diff
@@ -13,10 +13,7 @@ export function Bytes32TimeStamp(): number {
 }
 
 export function randUint16(): Uint16 {
-  return Math.floor(Math.random() * 2 ** 16)
-}
-export function randUint32(): Uint16 {
-  return Math.floor(Math.random() * 2 ** 32)
+  return Math.floor(Math.random() * (2 ** 16 - 1))
```

Comment:
@ScottyPoi I'm thinking we need something lower than just 2**16 -1 as our max, right?.  Isn't the issue that we could pick a sequence number for uTP too close to the maxium (2**16) and overflow when sending a long stream of packets (say a header accumulator that's 2mb and requires 2000 packets)?

Reply:
This is addressed in #331
I just used `2**15` as the max